cli/cmd/transformCmd.go

```diff
@@ -31,6 +31,7 @@ var (
 	}
 	schema string
 	input  string
+	ndjson bool
```

Comment:
thought we said changing `ndjson` to `stream`?

Reply:
Ah ok I changed the command option name but not in code, I'll make that change now